src/screens/coinDetails/screen/coinDetailsScreen.tsx

```diff
@@ -130,12 +129,13 @@ const CoinDetailsScreen = ({ navigation }: CoinDetailsScreenProps) => {
     }
 
     if (isPinCodeOpen) {
-      dispatch(
-        openPinCodeModal({
+      navigate({
+        routeName: ROUTES.SCREENS.PINCODE,
+        params:{
           navigateTo,
           navigateParams,
-        }),
-      );
+        }
+      })
     } else {
       navigate({
         routeName: navigateTo,
```

Comment:
Here

Reply:
The navigate service is also utilising compat package, that is why routeName is kept here as well.
![Screenshot 2022-09-08 at 4 31 54 PM](https://user-images.githubusercontent.com/6298342/189111778-4277d3e5-9024-44ea-b1d4-285342a8171a.png)